src/renderer/ssf-api.ts

```diff
@@ -244,9 +244,10 @@ export class SSFApi {
    * Method that returns various version info
    */
   public getVersionInfo(): Promise<IVersionInfo> {
-    const appName = remote.app.getName();
-    const appVer = remote.app.getVersion();
-    const cpuArch = os.arch() || '';
+    // TODO: get these values from a single source
+    const appName = 'Symphony'; // remote.app.getName();
+    const appVer = '14.0.0'; // remote.app.getVersion();
```

Comment:
[Proposal]
As ```package.json``` is a json, maybe we can read the correct version from there?
Same idea for the appName.

Reply:
Yes, that's a good idea will see if tha is possible